Scripts/DCS-BIOS/test/AircraftTest.lua

```diff
@@ -80,7 +84,7 @@ function TestAircraft:validateControlNames(module_name, documentation)
 			--   start with a letter
 			--   end with a letter or number
 			--   contain only letters, numbers, and underscores
-			local identifier_pattern = "^%u[%u%d_]*[%u%d]$"
+			local identifier_pattern = "^[%u%d_]*[%u%d]$"
```

Comment:
I'd make this change in a separate PR so we can fix the build right away and it doesn't depend on this PR, which will require a bit more work